src/adagrams.js

```diff
@@ -1,15 +1,119 @@
+// import { toPrimitive } from "core-js/fn/symbol";
+
+const LETTERPOOL = [
+  'A','A','A','A','A','A','A','A','A',
+  'B','B',
+  'C','C',
+  'D','D','D','D',
+  'E','E','E','E','E','E','E','E','E','E','E','E',
+  'F','F',
+  'G','G','G',
+  'H','H',
+  'I','I','I','I','I','I','I','I','I',
+  'J',
+  'K',
+  'L','L','L','L',
+  'M','M',
+  'N','N','N','N','N','N',
+  'O','O','O','O','O','O','O','O',
+  'P','P',
+  'Q',
+  'R','R','R','R','R','R',
+  'S','S','S','S',
+  'T','T','T','T','T','T',
+  'U','U','U','U',
+  'V','V',
+  'W','W',
+  'X',
+  'Y','Y',
+  'Z'
+]
+
 export const drawLetters = () => {
-  // Implement this method for wave 1
+  let letters = [];
+  let letterPoolCopy = [...LETTERPOOL];
+  while (letters.length < 10) {
+    const random = Math.floor(Math.random() * letterPoolCopy.length);
+    letters.push(letterPoolCopy[random]);
+    letterPoolCopy.splice(random, 1);
+  }
```

Comment:
Great approach, since you don't reassign the `letters` variable, use const instead